TreeDB/docs/evidence/vector-partition-revalidation-4093/tools/run_revalidation.py

```
@@ -0,0 +1,323 @@
+                  binary: Path, rebinder: Path, run_dir: Path) -> None:
+    network = f"gomap4093-r{repetition}"
+    names = [f"{network}-{group}" for group in GROUPS]
+    endpoints = {group: f"{ip}:{GROUP_PORT}" for group, ip in zip(GROUPS, CONTAINER_IPS)}
+    configs = [configure(root, run_dir, provenance, "container", repetition, index, endpoints,
+                         f"0.0.0.0:{PUBLIC_PORT}", rebinder) for index in range(4)]
+    topology_path = run_dir / "topology.json"
+    base.run([str(binary), "system-check-topology", "-configs", ",".join(map(str, configs)), "-out", str(topology_path)])
+    docker("network", "create", "--internal", "--subnet", "172.30.93.0/24", network)
+    started: list[str] = []
+    try:
+        for index, (config, name, ip) in enumerate(zip(configs, names, CONTAINER_IPS)):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)